moveit_planners/trajopt/config/setup.yaml

```diff
@@ -0,0 +1,14 @@
+trajopt_param:
```

Comment:
please add comments explaining what each of these parameters do, so that reviewers can better understand this PR

Reply:
You may want to cite the publication / note the paper's notation in the comments

Reply:
yaml file is the right place to put the citation or it should be in another file?

Reply:
sorry, i didn't mean cite the paper, i meant explain how these parameters relate to what were presented in the original research, since the var names probably differ